packages/admin-ui/src/index.js

```diff
@@ -14,10 +14,15 @@ if (process.env.NODE_ENV !== 'production' && process.env.REACT_APP_AXE) {
 
 // Fetch routes provided by Drupal modules.
 // This happens outsite of React and before the application starts.
-api('admin_ui_routes').then(({ routes }) => {
-  ReactDOM.render(
-    <App serverRoutes={routes} />,
-    document.getElementById('root'),
-  );
-});
-registerServiceWorker();
+api('admin_ui_routes')
+  .catch(err => {
+    console.err(err); // eslint-disable-line no-console
```

Comment:
Would it be possible to use the error message interface to present something to the user? I don't think this is critical though.

Reply:
@mattgrill I guess we could initialize the application with an error?

Reply:
Lets make a followup for that. Probably some more involved changes there.

Reply:
Sure: https://github.com/jsdrupal/drupal-admin-ui/issues/600 is the followup.